manifest/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+authors = ["o2sh <ossama-hjaji@live.fr>"]
+name = "onefetch-manifest"
+version = "0.1.0"
```

```suggestion
version = "2.13.2"
```
What do you think about keeping the version of `onefetch-remote` and `onefetch` the same?

This is a pattern I see in a lot of NPM workspaces/monorepos (jest, typescript-eslint, playwright). I think the gains are
- only one release/changelog is needed for the whole workspace
- it's honestly just easier than managing semver for each individual crate
- we signal that this sub-crate is tied to the main crate, and, while the library is publicly available, it is not stable independently of onefetch itself

Although, a con is that, as a user of some of these monorepos, it can be annoying when you get alerted of a new version yet nothing has changed in the package you're using.

---

if keeping the same version throughout the workspace means less maintenance on our part,  we should go for it.
Thanks @spenserblack 👍